src/Pages/AboutPage.js

```diff
@@ -0,0 +1,269 @@
+import React from 'react'
+import '../Styles/AboutPage.css'
+import VBImage from '../Assets/Images/vb-image.png'
+import EVImage from '../Assets/Images/evolution-of-a-vision-image.png'
+import Navbar from '../Components/Navbar'
+import SectionHeader from '../Components/SectionHeader'
+import SECard from '../Components/SECard'
+import EVCard from '../Components/EVCard'
+import VACard from '../Components/VACard'
+import Button from '../Components/Button'
+import SectionBreakLine from '../Components/SectionBreakLine'
+import Footer from '../Components/Footer'
+import DesignCardImg from '../Assets/Images/palette-icon.svg'
+import DevCardImg from '../Assets/Images/brackets-icon.svg'
+import BootstrapIcon from '../Assets/Images/bootstrap-icon.svg'
+import CanvaIcon from '../Assets/Images/canva-icon.svg'
+import CodepenIcon from '../Assets/Images/codepen-icon.svg'
+import FigmaIcon from '../Assets/Images/figma-icon.svg'
+import GitIcon from '../Assets/Images/git-icon.svg'
+import GithubFilledIcon from '../Assets/Images/github-filled-icon.svg'
+import JqueryIcon from '../Assets/Images/jquery-icon.svg'
+import MaterialUIIcon from '../Assets/Images/material-ui-icon.svg'
+import NodeJSIcon from '../Assets/Images/nodejs-icon.svg'
+import NPMIcon from '../Assets/Images/npm-icon.svg'
+import ReactIcon from '../Assets/Images/react-icon.svg'
+import VSCodeIcon from '../Assets/Images/vscode-icon.svg'
+import PhotoshopIcon from '../Assets/Images/photoshop-icon.svg'
+import XDIcon from '../Assets/Images/xd-icon.svg'
+import CassetteTape from '../Assets/Images/tape-icon.svg'
+import SewingMachine from '../Assets/Images/sewing-machine-icon.svg'
+import Recliner from '../Assets/Images/recliner-icon.svg'
+import StationWagon from '../Assets/Images/station-wagon-icon.svg'
+import { Link } from "react-router-dom"
+
+function AboutPage() {
+  return (
+    <div>
+      <Navbar />
+
+      <div className="vision-began-container">
+        <div className="vb-header">
+            <SectionHeader 
+            text="How The Vision Began"
+            />
+        </div>
+
+        <div className="vb-text-content">
+          <p>Ever had one of those lightning-bolt-from-the-sky moments? I did. 
```

Comment:
Inconsistent indentation in this whole file makes it difficult to read. Clean it up.